providers/hns/hns_roce_u.h

```diff
@@ -263,6 +268,17 @@ struct hns_roce_u_hw {
 	struct verbs_context_ops hw_ops;
 };
 
+/*
+ * The entries's buffer should be aligned to a multiple of the hardware's
+ * minimum page size.
+ */
+#define hr_hw_page_align(x)		align(x, HNS_HW_PAGE_SIZE)
+
+static inline unsigned int to_hr_hem_entries_size(int count, int buf_shift)
+{
+	return hr_hw_page_align(count << buf_shift);
```

Comment:
The only place the macro hr_hw_page_align is used is here, just call align() directly

Reply:
Thank you, I have fixed it.